tests/e2e/tests/generate/module/module-routing.ts

```diff
@@ -0,0 +1,22 @@
+import {join} from 'path';
+import {ng} from '../../../utils/process';
+import {expectFileToExist} from '../../../utils/fs';
+
+
+export default function() {
+  const moduleDir = join(process.cwd(), 'src', 'app', 'test-module');
```

Comment:
avoid using `process.cwd()`. And you can simply use `src/app/test-module` as it doesn't matter if it's windows or not (for the expect functions).